example-api-config/example-api-ingress-nginx.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      phantom_token on;
+      phantom_token_client_credential api-gateway-client Password123;
+      phantom_token_introspection_endpoint curity;
+      phantom_token_scopes read;
+      error_log  /var/log/nginx/error.log  debug;
```

Comment:
Having debug is very useful for developing but requires a debug-build of nginx, if I'm not mistaken. I suggest not to include a debug build or debug logs in this example but use a different log level as this is more in line with best practices.